src/Core/Settings.h

```diff
@@ -667,7 +667,7 @@ class IColumn;
     M(Bool, database_replicated_allow_only_replicated_engine, false, "Allow to create only Replicated tables in database with engine Replicated", 0) \
     M(Bool, database_replicated_allow_replicated_engine_arguments, true, "Allow to create only Replicated tables in database with engine Replicated with explicit arguments", 0) \
     M(DistributedDDLOutputMode, distributed_ddl_output_mode, DistributedDDLOutputMode::THROW, "Format of distributed DDL query result, one of: 'none', 'throw', 'null_status_on_timeout', 'never_throw'", 0) \
-    M(UInt64, distributed_ddl_entry_format_version, 5, "Compatibility version of distributed DDL (ON CLUSTER) queries", 0) \
+    M(UInt64, distributed_ddl_entry_format_version, 6, "Compatibility version of distributed DDL (ON CLUSTER) queries", 0) \
```

Comment:
Let's either change the PR category to Backward Incompatible Change and add a clear upgrade note to the changelog entry (see https://github.com/ClickHouse/ClickHouse/pull/42869#issuecomment-1311785870) or keep version 5 by default for now and use proper configs for tests

Reply:
which way is better?

Reply:
ok, then I'll close this PR and change test configs in https://github.com/ClickHouse/ClickHouse/pull/56383.